### PR TITLE
Changing_Buttons in Home, UserList files

### PR DESCRIPTION
**Pull request by TomekPaluch**

### CODE REVIEW

File: django_react_proj/users-fe/src/components/UserList/UserList.js [hunk 1]
```diff
@@ -32,7 +32,9 @@ const Actions = ({ user, resetState }) => {
   return (
     <>
       <TaskButton setDoneState={setDoneState} />
+      <> </>
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/UserList/UserList.js:35]: What does it to?

File: django_react_proj/users-fe/src/components/TaskButtonModal/TaskButtonModal.module.css [hunk 2]
```diff
@@ -23,3 +23,8 @@
     min-width: 20px;
     border-radius: 4px;
 }
+.inbtn{
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/TaskButtonModal/TaskButtonModal.module.css:26]: You can move it to some e.g. `common.css` and reuse it in those components 

`import commonStyles from '[..]/common.css`

File: django_react_proj/users-fe/src/components/Home/Home.js [hunk 3]
```diff
@@ -39,7 +63,9 @@ const Home = ({ getUserForView, setCurrentDate, currentDate, users, getUsers })
             <Calendar fullscreen={false} onChange={(date) => setCurrentDate(date.toDate())} />
           </Modal>
         </Col>
+        <Col>{currentDate.toDateString().substring(3)}</Col>
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/Home/Home.js:66]: You can use format from moment if you have it imported anyway

**Review by bakowroc — APPROVED** [review]

File: django_react_proj/users-fe/src/components/UserList/UserList.js (hunk 1 above)
Comment (TomekPaluch) [reply django_react_proj/users-fe/src/components/UserList/UserList.js:35]: It creates mini space between each button

File: django_react_proj/users-fe/src/components/Home/Home.js (hunk 3 above)
Comment (TomekPaluch) [reply django_react_proj/users-fe/src/components/Home/Home.js:66]: Done

**Pull request merged**